draft-xyz-rats-psa-endorsements.md

```diff
@@ -131,94 +141,173 @@ firmware measurement associated with Implementation ID
 
-The IAK public key is encoded as a COSE Key according to Section 7 of
-{{!RFC8152}} and wrapped in the `COSE_KeySet`.  The number of items in the
-`COSE_KeySet` MUST be 1.
+~~~ goat
+{::include art/atomic.txt}
+~~~
+{: #fig-atomic title="Atomic Provisioning Model"}
 
-The example in {{ex-identity-claim}} shows a CoMID identity claim carrying a
-secp256r1 EC public IAK associated with Instance ID `4ca3...d296`.
+A limitation of this model is that it may not be able to distinguish between a valid software RoT ( i.e. an
+expected combination of software component versions) from an invalid one.
 
+### Bundled
+{: #sec-bundled}
+In the bundled provisioning model all firmware components associated with a hardware RoT are grouped together in a single
+CoMID tag. The CoMID tag contains the identifier for the platform RoT associated to the PSA device. CoMID tag has a Reference Value Map which is used to carry a list of Reference Values ({{sec-ref-values}}). Each entry in the list carries the measurements associated to a firmware component. This model is applicable if the device firmware is made of separately measurable components however these components are always installed together in a single firmware update operation. In this model, the evolution of the versions of firmware component is collectively tracked using the CoMID updates and patches relations.
+
+~~~ goat
+{::include art/bundled.txt}
 ~~~
-{::include examples/instance-pub.diag}
+{: #fig-bundled title="Bundled Provisioning Model"}
+
+Each tagged edge is the CoMID-to-CoMID link relation describing an
+update or patch action on the target node by the source node.
+
+### Firmware Updates and Patches
+
+{: #sec-fw-evo}
+
+Firmware measurements that are part of the same upgrade chain can be linked
+together using one of `comid.patches` or `comid.updates` relations, depending on
+the precise nature of their relationship.  For example, if using semantic
+versioning {{SEMA-VER}}, a bump in the MINOR version number would be associated
+with a `comid.updates` relation, whereas an increase in the PATCH indicator
+would have a corresponding `comid.patches` relation between the involved
+components.  Note that `comid.updates` relations would only occur between
+firmware measurements that have PATCH number 0 and "adjacent" MINOR numbers
+as illustrated in {{fig-updates-patches}}.  Note that changing the MAJOR
+version number would typically result in a separate product / upgrade chain.
+
+~~~ goat
+{::include art/updates-patches.txt}
 ~~~
-{: #ex-identity-claim title="Example Identity Claim"}
+{: #fig-updates-patches title="Updates, Patches Relations and Semantic Versioning"}
 
-### CoMID Layout
+The Reference Value CoMID of the patching or updating firmware has a
+`linked-tag-entry` map populated as follows:
 
-An Identity CoMID can carry as many Identity Claims as needed.
+* key 0 contains the tag identifier of the Reference Value CoMID of the patched
```

Comment:
instead of key 0 and key 1 we should use their names

Reply:
Agree!